frontend/workflows/projectSelector/src/project-links.tsx

```diff
@@ -47,23 +46,33 @@ const StyledMoreVertIcon = styled.span({
 
 const StyledLinkTitle = styled.span({
   fontWeight: "bold",
+  padding: "7px 0px 7px 0px",
```

Comment:
nit: you can simplify this css by using `7px 0` which will apply 7px to the top and bottom and 0 to the left and right